examples/with-typescript-graphql/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["next/babel"],
-  "plugins": ["graphql-let/babel"]
+  "plugins": []
 }
```

Comment:
if no special plugins are needed anymore, I recommend removing `.babelrc` to leverage SWC for builds. :+1:

Reply:
@balazsorban44 will do, thanks!